app/controllers/people_controller.rb

```diff
@@ -16,7 +16,10 @@ def show
         response.headers['Content-Length'] = @person.v_card.size.to_s
       }
       format.pdf {
-          pdf = Services::Token.generate_token_pdf(@person, settings.token_media)
+          Rails.logger.info "settings #{settings.token_media}"
```

Comment:
application_controler.rb publishes Rails.application.config as settings. token_media is currently a global settings but could be moved into service_provider